lms/djangoapps/certificates/migrations/0020_certificatehtmlviewconfiguration_data.py

```diff
@@ -55,7 +55,10 @@ def forwards(self, orm):
         )
 
     def backwards(self, orm):
-        "Write your backwards methods here."
+        """
+        Rolling back to zero-state, so remove all currently-defined configurations
+        """
+        configurations = orm.CertificateHtmlViewConfiguration.objects.all().delete()
```

Comment:
No need to assign to a variable here, and I believe it's a pylint violation. Just take the `configurations` variable out.